lib/index.d.ts

```diff
@@ -14,10 +14,20 @@ export * from "./common";
 export * from "./joint";
 export * from "./shape";
 export * from "../testbed";
+  
+export interface Manifold {
+  type: number;
```

Comment:
Maybe this is a bit cleaner if you define an enum for the manifold type, but do it in whatever way you want.

Reply:
done.